tools-local/tasks/core-setup.tasks.sln

```diff
@@ -1,22 +0,0 @@
-﻿
```

Comment:
Why is this deleted?

Reply:
this folder will be migrated into the runtime root repo. To reduce noise with the sln (opening it in different VS versions will update its version) and as the solution only contains one project, I removed it.